examples.manifest.yaml

```diff
@@ -20,13 +20,43 @@
     port: 9009
 - name: ilp-auth-tls
   lang: go
-  path: examples/auth-and-tls/main.go
+  path: examples/tcp/auth-and-tls/main.go
   header: |-
-    Go client library [docs](https://pkg.go.dev/github.com/questdb/go-questdb-client/v2)
+    Go client library [docs](https://pkg.go.dev/github.com/questdb/go-questdb-client/v3)
     and [repo](https://github.com/questdb/go-questdb-client).
   auth:
     kid: testUser1
     d: 5UjEMuA0Pj5pjK8a-fa24dyIf-Es5mYny3oE_Wmus48
   addr:
     host: localhost
     port: 9009
+- name: http
```

Comment:
nit: I'd add `ilp-` suffix to the example names: `ilp-http`, `ilp-http-auth`, `ilp-http-auth-tls`